platform/kube/controller.go

```diff
@@ -51,6 +51,11 @@ type cacheHandler struct {
 	handler  *chainHandler
 }
 
+const (
+	// ServiceSuffix is the hostname suffix used by a Kubernetes service
+	ServiceSuffix = "svc.cluster.local"
```

Comment:
Nit: May be add a todo here to make this configurable in future? I vaguely remember having the ability to define the suffix when initializing the kubernetes cluster. If that has changed , then please disregard this comment.